tests/conftest.py

```diff
@@ -2521,6 +2521,12 @@ def raredisease_params_file_path(raredisease_dir, raredisease_case_id) -> Path:
     ).with_suffix(FileExtensions.YAML)
 
 
+@pytest.fixture(scope="function")
+def raredisease_gene_panel_path(raredisease_dir, raredisease_case_id) -> Path:
```

Comment:
I don't see this fixture used anywhere. If it not part of another PR consider removing it